internal/converter/internal/otelcolconvert/otelcolconvert.go

```diff
@@ -291,28 +291,39 @@ func validateNoDuplicateReceivers(groups []pipelineGroup, connectorIDs []compone
 
 func buildConverterTable(extraConverters []ComponentConverter) map[converterKey]ComponentConverter {
 	table := make(map[converterKey]ComponentConverter)
-	allConverters := append(converters, extraConverters...)
+
+	// Ordering is critical here because conflicting converters are resolved with
+	// the first one in the list winning.
+	allConverters := append(extraConverters, converters...)
```

Comment:
From our discussion, I implemented a first in wins in case of overlap with a priority for injected converters. We have no overlap today so this is a safeguard for now.